supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkBackendFactory.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend.impl.spark
+
+import akka.actor.{ActorRef, Props}
+import cromwell.backend._
+import cromwell.backend.sfs.SharedFileSystemExpressionFunctions
+import cromwell.backend.io.JobPaths
+import cromwell.core.CallContext
+import wdl4s.Call
+import wdl4s.expression.WdlStandardLibraryFunctions
+
+case class SparkBackendFactory(configurationDescriptor: BackendConfigurationDescriptor) extends BackendLifecycleActorFactory {
+  override def workflowInitializationActorProps(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], serviceRegistryActor: ActorRef): Option[Props] = {
+    Option(SparkInitializationActor.props(workflowDescriptor, calls, configurationDescriptor, serviceRegistryActor))
+  }
+
+  override def jobExecutionActorProps(jobDescriptor: BackendJobDescriptor, initializationData: Option[BackendInitializationData],
+                                      serviceRegistryActor: ActorRef): Props = {
+    SparkJobExecutionActor.props(jobDescriptor, configurationDescriptor)
```

Comment:
Interesting that the `serviceRegistryActor` isn't passed to the execution actor, which means no metadata is being published during execution.